build.ps1

```diff
@@ -59,7 +59,7 @@ if ($pack) {
         ($outputHashingIndex+1 -ge $args.Count) -Or
         ($args[$outputHashingIndex+1].ToLower() -ne "all")

```

Comment:
can you remove this .ToLower() as well

Reply:
@shhsu done.

Reply:
actually, I just realized, `-ne` does not work here at all because it is not case sensitive, can you use `.equals()` instead

Reply:
Fixed.